ambry-store/src/main/java/com.github.ambry.store/BlobStore.java

```diff
@@ -19,6 +19,7 @@
 import com.github.ambry.config.StoreConfig;
 import com.github.ambry.utils.FileLock;
 import com.github.ambry.utils.Time;
+import com.github.ambry.utils.Utils;
```

Comment:
sorry for the back and forth but now that the validation check is all inside the store, I think we should move this function to this class as a private function.